### ISSUE TITLE
We desperately need a notch function

### ISSUE THREAD

**Issue by Toontje** [cjcliffe/CubicSDR#686]
![screenshot 2018-10-21 at 13 52 20](https://user-images.githubusercontent.com/4884243/47266554-9d600280-d538-11e8-9322-822c813e5335.jpg)

**Comment by cjcliffe** [comment]
@Toontje agreed; I started working on filtering last year but need to go back and rework some UI and do cleanup before it can be added properly.

**Comment by cjcliffe** [comment]
Relevant liquid-dsp feature appears to have been added recently: https://github.com/jgaeddert/liquid-dsp/commit/f0f1b1bf86088abb9835cc852f1265a5f6ee3741
